hooks/pre/35-change-evr-extension-ownership.rb

```diff
@@ -0,0 +1,26 @@
+# In Katello 4.16, the 'evr' extension is removed from PostgreSQL and integrated into the Katello database via a migration.
+# This hook ensures the 'evr' extension's ownership is transferred to the 'foreman' user so migrations can act on it.
+
+if local_postgresql? && execute("rpm -q postgresql-evr", false, false)
+  is_postgresql_active = execute_command("systemctl is-active postgresql", false, true)&.last&.exitstatus == 0
```

Comment:
I'm debating simplification and wonder what you think. And I'm only talking about the `op` "mode", not about `noop`.

You can always call `start_services()` even if it was already running, so the check really is only useful for stopping. But do we need to stop the service? The installer will later start it anyway. Perhaps just leave it running?

Reply:
Hmm..The idea was to leave the service in the state this hook finds it in..I don't feel strongly about leaving it running after this hook cause it would get stopped by anything after this that needs it stopped. I'll push to simplify the thing..